adb_graphics/datahandler/gribfile.py

```diff
@@ -73,6 +73,8 @@ def free_fcst_names(self, ds, fcst_type):
 
         ret = {}
 
+        fhr = self.coord_dims['fcst_hr'][-1]
```

Comment:
This line is causing the ensemble plotting to fail, with the following messages in the log file
```
^MTraceback (most recent call last):
  File "/mnt/lfs1/BMC/wrfruc/chunhua/code/ufs-srweather-app.0222/python_graphics/create_graphics.py", line 725, in <module>
    graphics_driver(CLARGS)
  File "/mnt/lfs1/BMC/wrfruc/chunhua/code/ufs-srweather-app.0222/python_graphics/adb_graphics/utils.py", line 329, in wrapper_timer
    value = func(*args, **kwargs)
  File "/mnt/lfs1/BMC/wrfruc/chunhua/code/ufs-srweather-app.0222/python_graphics/create_graphics.py", line 638, in graphics_driver
    gribfiles = gribfile.GribFiles(
  File "/mnt/lfs1/BMC/wrfruc/chunhua/code/ufs-srweather-app.0222/python_graphics/adb_graphics/datahandler/gribfile.py", line 59, in __init__
    self.contents = self._load()
  File "/mnt/lfs1/BMC/wrfruc/chunhua/code/ufs-srweather-app.0222/python_graphics/adb_graphics/datahandler/gribfile.py", line 196, in _load
    renaming = self.free_fcst_names(dataset, fcst_type)
  File "/mnt/lfs1/BMC/wrfruc/chunhua/code/ufs-srweather-app.0222/python_graphics/adb_graphics/datahandler/gribfile.py", line 76, in free_fcst_names
    fhr = self.coord_dims['fcst_hr'][-1]
KeyError: 'fcst_hr'
+ print_err_msg_exit 'Call to pyscript "JREGIONAL_RUN_PYTHON_GRAPHICS" failed.'

```
Removing this line makes both the ensemble and deterministic plotting work. 
I am not sure if this is the right fix for the ensemble plotting, given it was added for a purpose. 
What do you think?